book/src/tooling/coap.md

```diff
@@ -0,0 +1,248 @@
+# CoAP
+
+> **Warning**: This documentation is currently ahead of the implementation roadmap.
+
+[CoAP] is an **application layer protocol similar to HTTP** in its use
+(you can POST data onto a resource on a server that is identified by a URL)
```

Comment:
Would be "for example, " a sufficient addition? I don't want to go into exhaustively listing what it can do, but just to give an example in how they are alike.

Reply:
Turned into an "eg. you could POST".